src/index.js

```diff
@@ -1,107 +1,54 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
-import * as dpRender from 'datapackage-render'
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
-const { Dataset } = require("data.js")
-const toArray = require("stream-to-array")
-
-const instances = document.getElementsByClassName('react-me-datapackage-views')
-function parseDatapackageIdentifier(stringOrJSON) {
-  try {
-    return JSON.parse(stringOrJSON)
-  } catch (e) {
-    return stringOrJSON
-  }
-}
-
-for (const instance of instances) {
-  const DP_ID = parseDatapackageIdentifier(instance.getAttribute('data-datapackage-json'))
-
-  // Load Dataset object
-  Dataset.load(DP_ID).then(async (dataset) => {
-    const tabularFormats = ['csv', 'tsv', 'dsv', 'xls', 'xlsx']
-    // TODO: support local files
-    // Data fetcher
-    dataset.resources.forEach(async (file) => {
-      if (file.displayName === 'FileInline') {
-        return
-      } else if (file.descriptor.path && file.descriptor.path.includes('datastore_search')) {
-        // Datastore, e.g., when a path is a 'datastore_search' API
-        const response = await fetch(file.descriptor.path)
-        if (!response.ok) {
-          file.descriptor.unavailable = true
-          return
-        }
-        const result = await response.json()
-        file.descriptor.data = result.result.records
-      } else if (file.displayName === "FileRemote" && tabularFormats.includes(file.descriptor.format)) {
-        // Tabular data
-        try {
-          const rowStream = await file.rows({size: 100, keyed: true})
-          const data = await toArray(rowStream)
-          if (data.length > 0) {
-            file.descriptor.data = data // This makes it FileInline
-          } else {
-            file.descriptor.unavailable = true
-          }
-        } catch (e) {
-          console.log(e)
-          file.descriptor.unavailable = true
-        }
-      } else if (file.descriptor.format.toLowerCase().includes('json')) {
-        // Geographical data
-        const response = await fetch(file.descriptor.path)
-        if (!response.ok) {
-          file.descriptor.unavailable = true
-          return
+const datapackage = {
```

Comment:
I don't think we need to do this here. We can check this using cosmos or unit tests.